### PR TITLE
Convert GitHub Actions workflow to new format

### PR DESCRIPTION
**Pull request by ScottBrenner**
[HCL workflows are no longer being run](https://github.blog/changelog/2019-10-01-github-actions-hcl-workflows-are-no-longer-being-run/), this PR converts this repo's GitHub Actions workflow to the new YAML format.

Proposed workflow copied from https://github.com/marketplace/actions/ansible-lint#usage.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      with:
+        # [required]
+        # Paths to ansible files (i.e., playbooks, tasks, handlers etc..)
+        # or valid Ansible directories according to the Ansible role
+        # directory structure.
+        # If you want to lint multiple ansible files, use the following syntax
+        # targets: |
+        #   playbook_1.yml
+        #   playbook_2.yml
+        targets: "wordpress-nginx/site.yml"
```
Comment (ScottBrenner) [inline .github/workflows/main.yml:26]: From https://github.com/ansible/ansible-examples/blob/6e7dd3d3613e1448703a2137ff892c2da6196e97/.github/main.workflow#L9

**Comment by ScottBrenner** [comment]
@webknjaz Mind reviewing?

File: .github/workflows/main.yml (hunk 1 above)
Comment (webknjaz) [reply .github/workflows/main.yml:26]: I think the linter offenses should be fixed before merging this

Comment (webknjaz) [reply .github/workflows/main.yml:26]: Also, YAML doesn't need quotes
```suggestion
        targets: wordpress-nginx/site.yml
```

Comment (ScottBrenner) [reply .github/workflows/main.yml]: > the linter offenses should be fixed before merging this
Opened https://github.com/ansible/ansible-examples/issues/271

Comment (ScottBrenner) [reply .github/workflows/main.yml]: @webknjaz Opened https://github.com/ansible/ansible-examples/pull/278 to fix linter offenses!

**Comment by webknjaz** [comment]
Looks like it's still failing

**Comment by ScottBrenner** [comment]
> still failing

Yes, will have to rebase once https://github.com/ansible/ansible-examples/pull/278 is merged.

**Comment by ScottBrenner** [comment]
Moved to #279.

**Pull request closed without merging**